plugins/chain_plugin/test/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_executable( test_account_query_db test_account_query_db.cpp )
+
+target_link_libraries( test_account_query_db chain_plugin eosio_testing)
+
+target_include_directories( test_account_query_db PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include" "${CMAKE_CURRENT_SOURCE_DIR}/../../../libraries/chain/include" "${CMAKE_CURRENT_SOURCE_DIR}/../../../libraries/chainbase/include")
```

Comment:
It should not be necessary to explicitly add these include directories.  linking `chain_plugin` which depends on `eosio_chain` which depends on `chainbase` should have brought all those paths in.  Does it not?

Reply:
> It should not be necessary to explicitly add these include directories. linking `chain_plugin` which depends on `eosio_chain` which depends on `chainbase` should have brought all those paths in. Does it not?

I got some build errors earlier without adding those include directories, but now the build errors are disappeared. Will remove the include directories from the code.

Reply:
The unnecessary include directories have been removed.